scripts/get-resource-index-ref-matrix

```diff
@@ -100,7 +109,15 @@ get_resource_index_at_commit() {
     # script. We can remove it once all the servers have been updated.
     #   -Jover, 01 May 2024
     if [[ -f "$repo/scripts/get-resource-index.js" ]]; then
```

Comment:
Thoughts on dropping this conditional while we're here? Would make debugging simpler and us more confident in the code which was executed

Reply:
Or, alternatively, just use the `jq` code which seems much simpler to reason with to my eyes. I'm presuming there's some reason you (pl) wanted to use a custom script for this?